src/hidimstat/statistical_tools/lasso_test.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+from sklearn.linear_model import LassoCV
+from sklearn.model_selection import KFold
+
+
+def preconfigure_LassoCV(estimator, X, X_tilde, y, n_alphas=20):
```

Comment:
```suggestion
def preconfigure_lasso_path(estimator, X, X_tilde, y, n_alphas=20):
```
I think it's good naming practice to use snake_case for functions, CamelCase for classes.